src/test/java/org/primeframework/mvc/util/QueryStringToolsTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2024, Inversoft Inc., All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+package org.primeframework.mvc.util;
+
+import java.util.List;
+import java.util.Map;
+
+import org.testng.annotations.Test;
+import static org.testng.Assert.assertEquals;
+
+/**
+ * @author Daniel DeGroff
+ */
+public class QueryStringToolsTest {
+  @Test
+  public void parseQueryString() {
```

Comment:
If you're adding a test, I recommend also testing
1. a query string parameter without a value
2. a query string parameter with multiple values